src/transformers/pipelines/audio_utils.py

```diff
@@ -31,10 +31,14 @@ def ffmpeg_read(bpayload: bytes, sampling_rate: int) -> np.array:
     ]
 
     try:
-        with subprocess.Popen(ffmpeg_command, stdin=subprocess.PIPE, stdout=subprocess.PIPE) as ffmpeg_process:
+        with subprocess.Popen(
+            ffmpeg_command, stdin=subprocess.PIPE, stdout=subprocess.PIPE
+        ) as ffmpeg_process:
```

Comment:
nit - formatting change shouldn't be applied here 

```suggestion
        with subprocess.Popen(ffmpeg_command, stdin=subprocess.PIPE, stdout=subprocess.PIPE) as ffmpeg_process:
```